### ISSUE TITLE
[Request] option to generate chunk/regions in a world but filled with air

### ISSUE THREAD

**Issue by AsierBR** [Querz/mcaselector#104]
So i dont really know if this goes here, sorry if this is an unproffesional request but i think that this feature could help a lot of people that just want to play skyblock maps. right now i have found the hard way that there are not really good ways to make a void world that has biomes anymore.

what im asking basically is:
-an option after you select a region/s or chunk/s that lets you generate all those chunk but with only air blocks
-should respect what biome was originally there
-should also respect structures, like witch huts or ocean momuments

i know that this is not really mcaselector purpouse, but i think that is not too hard to implement since you can more or less already do it manually (by importing chunks that are void in other places), and since right now there is not really any other external tool that does it i think that i would be a celebrated new feature

thanks for reading, and again, sorry is this in uncalled for.

**Comment by AsierBR** [comment]
even without the structures having the option to have a void world with biomes easily in vanilla using an external tool is certainly a welcome addition, im certain that i would see use, specially for empyting the end and nether dimenions, and right now with 1.16 nether coming...

if you can add it im for sure going to use it, thanks for answering btw

**Comment by AsierBR** [comment]
hi again, i have seem that you updated the project with a commit that talks about deleting sections, does that have to do with this request? if i does, does it already work? i have tried to build the project myself but i cant because an error with missing jdk.

thansk for answering!

**Comment by Querz** [comment]
Indeed, this adds a new option to the "Change NBT" Tool to delete all sections from a chunk. I'm still waiting for a new release until Minecraft 1.16 is out (which seems to be sooner than later).

This feature will allow you to select existing chunks and delete all their sections, while keeping the biomes and structure boundaries intact, pretty simple actually. However, it does not generate empty chunks (yet).

Here's a compiled version of the current state of the master branch if you want to test it:
[mcaselector-1.10-pre.jar.zip](https://github.com/Querz/mcaselector/files/4634439/mcaselector-1.10-pre.jar.zip)

**Comment by AsierBR** [comment]
thanks a lot! i have been playing with it a little and it simply works. at first i tought that i was not doing anything since i selected quite a number of chunks and after selecting them for voiding i took very little time to process them. but later in the game they were deleted.

there is one "bug" with the map that is show when you load a world, at least it has happened to me, but it shows the world as it was before deleting the chunks, even after entering the world, visiting there, exiting minecraft and entering mcaselector again.

**Comment by Querz** [comment]
Ah yes, forgot that the visuals of the map actually change now with the new feature, it doesn't with the other options in the NBT Changer.
